masterlist.yaml

```
@@ -11295,13 +11295,30 @@ plugins:
   - name: 'OOOShiveringIsles.esp'
     url: [ 'https://www.nexusmods.com/oblivion/mods/46508/' ]
     tag:
+      - Actors.RecordFlags
+      - -Body-M
+      - C.Name
+      - C.Owner
+      - EffectStats
+      - EnchantmentStats
+      - Invent.Change
+      - NpcFacesForceFullImport
+      - R.Eyes
       - name: -R.Head
         condition: 'file("Oblivion_Character_Overhaul.esp")'
+      - R.Relations.Add
+      - -Relations
+      - Relations.Add
```

For removal of tags, you can ignore the deprecated ones.
But you can remove part of the deprecated tags behavior using the new tags.

```suggestion
      - Relations.Add
      - -Relations.Change
```